src/modules/flight_mode_manager/tasks/Auto/FlightTaskAuto.hpp

```diff
@@ -92,6 +92,13 @@ class FlightTaskAuto : public FlightTask
 	bool updateInitialize() override;
 	bool update() override;
 
+	bool setCruisingSpeed(const float cruising_speed_m_s) override
+	{
+		_mc_cruise_speed = cruising_speed_m_s;
+		_commanded_speed_ts = hrt_absolute_time();
+		return true;
```

Comment:
I would make it void until there's actually a path that can fail and the return value gets handled.
```suggestion
```